src/SFA.DAS.EmployerIncentives.ValueObjects/NewApprenticeIncentive.cs

```diff
@@ -7,6 +7,8 @@ namespace SFA.DAS.EmployerIncentives.ValueObjects
     public class NewApprenticeIncentive : ValueObject
     {
         private readonly DateTime EligibilityStartDate = new DateTime(2020, 8, 1);
+        private const decimal TwentyFiveOrOverIncentiveAmount = 2000;
```

Comment:
Can these const values be made public and used in the unit tests instead of magic numbers?

Reply:
Call me perverse but I'd actually like the test to break if these numbers are changed.

Reply:
agreed, we shouldn't need to change access just for tests.